velox/vector/tests/VectorMaker.h

```diff
@@ -416,18 +416,28 @@ class VectorMaker {
   /// nullable. Only null array elements are supported; not null arrays.
   template <typename T>
   ArrayVectorPtr arrayVectorNullable(
-      const std::vector<std::vector<std::optional<T>>>& data) {
+      const std::vector<std::optional<std::vector<std::optional<T>>>>& data) {
```

Comment:
This allows us to pass in null arrays too.. which is usefull if you want to create nested arrays which can have null arrays.